app/src/main/java/org/mozilla/fenix/components/metrics/MetricsStorage.kt

```diff
@@ -64,27 +77,87 @@ internal class DefaultMetricsStorage(
             Event.GrowthData.FirstUriLoadForDay -> {
                 settings.uriLoadGrowthLastSent = System.currentTimeMillis()
             }
+            Event.GrowthData.FirstWeekSeriesActivity -> {
+                settings.firstWeekSeriesGrowthSent = true
+            }
         }
     }
 
+    private fun updateDaysOfUse() {
+        val daysOfUse = settings.firstWeekDaysOfUseGrowthData
+        val currentDate = Calendar.getInstance()
```

Comment:
Should these Calendar instances also be using the same locale as the time format (Locale.US)?

```suggestion
        val currentDate = Calendar.getInstance(Locale.US)
```